js/main.js

```diff
@@ -67,4 +40,275 @@ const toggleMenu = ()=> {
+const workSection = document.querySelector('.work__section')
+const workBtn = document.querySelectorAll('.work__button')
+
+const createPopup = (ind) => {
+    const work = cards[ind]
+    const {
+        img,
+        alt,
+        name,
+        feature,
+        feature2,
+        feature3,
+        description,
+        language,
+        language2,
+        language3,
+        language4,
+        framework
+    } = work
+
+    const article = document.createElement('article')
+    article.classList.add('popup__description')
+    article.innerHTML = 
+    `
+    <header class="popup__header">
+        <h2 class="popup__title">${name}</h2> 
+        <i onclick="" class="fa-solid fa-times popup__xmark"></i>
+    </header>
+    <ul class="popup__features">
+        <li class="popup__features__item">${feature}</li>
+        <li class="popup__features__item">${feature2}</li>
+        <li class="popup__features__item">${feature3}</li>
+    </ul>
+    <figure class="popup__image__container multi">
+        <img src="${img}" alt="${alt}" class="popup__image"> 
+    </figure>
+    <div class="popup__features__container">
+        <p class="popup__paragraph">${description}</p>
+        <div class="popup__bottom__container">
+            <ul class="popup__list__container ruby">
+                <li class="popup__language__item">${language}</li>
+                <li class="popup__language__item">${language2}</li>
+                <li class="popup__language__item">${language3}</li>
+                <li class="popup__language__item">${language4}</li>
+                <li class="popup__language__item">${framework}</li>
+            </ul>
+            <div class="about__btn__container">
+                <button class="popup__button">
+                    See live
+                    <i class="fa-solid fa-arrow-up-right-from-square"></i>
+                </button>
+                <button class="popup__button">
+                    See source
+                    <i class="fa-brands fa-github"></i>
+                </button>
+            </div>
+        </div>    
+    </div>
+    `
+    popupWindow.appendChild(article)
+
+    document.body.style.overflow = 'hidden'
+    
+    const closePopup = document.querySelector('.popup__xmark')
+    closePopup.addEventListener('click', ()=> {
+        workSection.classList.remove('blurEffect')
+        navbarSection.style.display = 'flex'
+        document.body.style.overflow = 'scroll'
+        popupWindow.removeChild(article)
+        workBtn.forEach((btn) => {
+            btn.classList.remove('press')
+        })
+    })
+}
+
+// WORK BUTTONS    
+workBtn.forEach((btn) => {
+    btn.addEventListener('click', (e) => {
+        btn.classList.toggle('press')
+        const ind = e.target.id
+        createPopup(ind)
+        workSection.classList.add('blurEffect')
+        navbarSection.style.display = 'none'
+    })
+})
+
+// CONTACT FORM
+const form = document.getElementById('contact__form')
```

Comment:
- [ ] Well! I can see that you've made efforts to address the previous reviewer's update request. However, I noticed that when I fill out the form and click the submit button, if the submission is unsuccessful, the data is still being stored. 🔄 As per the project requirements, the data object should only be updated and saved to local storage when the user changes the content of any input field in the form not when click submit button.

Please ensure that your code follows the specified requirement and updates the data object and local storage only when there is a change in the form input fields.  Keep up the good work! 💪🌟